test/testng/org/openjdk/jextract/test/toolprovider/TestClassGeneration.java

```diff
@@ -186,9 +186,6 @@ public void testStructMember(String structName, MemoryLayout memberLayout, Class
         MemoryLayout structLayout = (MemoryLayout) layout_getter.invoke(null);
         try (Arena arena = Arena.ofConfined()) {
             MemorySegment struct = arena.allocate(structLayout);
-            Method vh_getter = checkMethod(structCls, memberName + "$VH", VarHandle.class);
```

Comment:
This removal comes from the fact that StructBuilder no longer relies on var handle generation in struct field getter/setters (we just generate the correct layout inline).